os/board/common/crashdump.c

```diff
@@ -238,6 +312,48 @@ static int send_fsdump(void)
 	return 0;
 }
 
+static int send_extfsdump(void)
+{
+	int i = 0;
+	int fd;
+	int index;
+	uint8_t *ptr;
+	size_t size;
+	char ext_userfs_partition[EXTUSERFS_PARTITION_NAME_SIZE + 1] = { '\0' };
+
+	/*Send ACK for EXTERNAL FSDUMP Handshake */
+	up_lowputc('A');
+
+	i = get_ext_userfs_part(&size, &index);
+	if (i < 0) {
+		return -1;
+	}
+
+	/* Send external userfs partition size to host */
+	ptr = (uint8_t *)&size;
+	for (i = 0; i < sizeof(size); i++) {
+		up_lowputc((uint8_t)*ptr);
+		ptr++;
+	}
+
+	/* If extracted size is 0, it menas ther eis no partition available to dump */
+	if (size == 0) {
+		return 0;
+	}
+
+	snprintf(ext_userfs_partition, EXTUSERFS_PARTITION_NAME_SIZE, "/proc/fs/smartfs/smart1p%d/dump_ext", index);
+
+	fd = open(ext_userfs_partition, O_RDONLY);
+	if (fd < 0) {
+		return -EIO;
+	}
+
+	/* Send dummy read request, no data will actually be returned. Procfs entry automatically sends data over tty */
+	i = read(fd, NULL, 63);
```

Comment:
If `i` is not used anymore, you don't need to assign `i` from `read`.
You can just call `(void)read(xxx);`

Reply:
Ahh ohkkay, Thankyou